.golangci.yml

```diff
@@ -19,8 +17,6 @@ linters-settings:
   importas:
     no-unaliased: true
     alias:
-      - pkg: connectrpc.com/connect
-        alias: connect
```

Comment:
This importas alias config didn't make sense after migrating the module to connectrpc.